tests/integration/high_availability/test_replication.py

```diff
@@ -32,16 +32,12 @@
 TIMEOUT = 15 * 60
 
 
-@pytest.mark.order(1)
 @pytest.mark.replication_tests
 async def test_build_and_deploy(ops_test: OpsTest) -> None:
     """Simple test to ensure that the mysql and application charms get deployed."""
     await high_availability_test_setup(ops_test)
 
 
-# TODO: change test order to '2' after the following issue is resolved
-# https://github.com/canonical/mysql-k8s-operator/issues/102
-@pytest.mark.order(3)
```

Comment:
I just checked the backlog, and the [ticket](https://warthogs.atlassian.net/browse/DPE-915) associated with the issue is still pending. I was going to suggest we move the test to the end of the file with a comment (so that we mark why the test runs last)

Reply:
That is surprising (since I do not know why the test would pass unless the issue is intermittent). We can consult Alex and possibly create a ticket to further investigate

Reply:
If test is happy, no need to report it as a ticket. Just double check the test is executed.
Our charms were actively developed and the situation might change.
The ticket https://warthogs.atlassian.net/browse/DPE-915 is still open, but now we clearly see Juju issues there while before it was a charm crash on Cluster.add_instance.

Reply:
>  Should I move the test back?

I think it should stay in the position it pass. If currently pass, keep it AS IS :-)

Reply:
As the test passed in all possitions, keep it AS IS. IMHO. Up 2 Shayan here.